client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -85,6 +87,8 @@ const jetpackStatsCommercialPaywall = [
 	STAT_TYPE_COMMENTS,
 	STATS_TYPE_DEVICE_STATS,
 	STATS_FEATURE_UTM_STATS,
+	STATS_FEATURE_LOCATION_REGION_VIEWS,
```

Comment:
Feels like we don't need these feature to be here as we've got them in `granularControlForJetpackStatsCommercialPaywall` already?

Reply:
Thanks! You are right. I just removed it :)